### ISSUE TITLE
BAG OF catalogue values are stored in Baseset basket of the catalogue

### ISSUE THREAD

**Issue by signedav** [opengisch/QgisModelBaker#754]
If we have BAG OF to catalogues.
```
    CLASS AssetItem =
      ManCatLabel : BAG {1..*} OF GeolAssetsCatalogues_V2.Catalogues.ManCatLabelRef;
```

A reference table is created with ili2db (called ManCatLabelRef). The entries there need to be assigned to a basket (FK not NULL) but it doesn't matter to which one. Because in the end they are exported / validated concerning their parent in the main data (AssetItem). 

So Model Baker takes the first available basket. And this is (usually) one of the Baseset (since it's created automatically) with the topic of the cataloge. All the baskets from the Catalogueset are filtered out. But actually those baskets that are an intersection between Baseset and Catalogue-topic are not really used (because Catalogue-data is in the Catalogueset). But if we remove the creation of it (https://github.com/opengisch/QgisModelBaker/issues/599#issuecomment-1091145098), we need to care for this case here.

**Comment by signedav** [comment]
TL;DR: After some research I came to the conclusion, that this should not bother us.

The ManCatLabelRef in the example above is designed in the topic of the catalogue and with it, it takes per default the basket of the catalogue data (according to the topic). This seems to be wrong, but it doesn't matter. It's a STRUCTURE in the end and so it's data belongs anyway to it's proprietor (the CLASS referencing to it). So when I export the Basket of the Catalogue those data are not affected, and when I export the data of the CLASS referencing to it, those date are contained...

I updated the documentation concerning that.
